README.md

```diff
@@ -215,6 +226,27 @@ The schema for this type is as follows:
 }
 ```
 
+### Generating an external manifest
+
+The `-s` option puts the manifest in an external sidecar file in the same location as the output file. The
+manifest will have the same output filename but with a ".c2pa" extension. The output file will be untouched. 
+
+```shell
+c2patool -s sample/test.json -o new_manifest.jpg
+```
+
+### Generating a remote manifest
+
+The `-r` option places an http reference to manifest in the output file. The manifest is returned as an external sidecar file in the same location as the output file. The manifest will have the same output filename but with a ".c2pa" extension. The manifest should then be placed at the location specified by the `-r` option. When using remote manifests the remote URL should be publicly accessible to be most useful to users. When verifying an asset, remote manifests are automatically fetched. You can disable this by removing feature `fetch_remote_manifests` from Cargo.toml.
+
+```shell
+c2patool -r http://my_server/myasset.c2pa sample/test.json -o new_manifest.jpg
+```
+
+In the example above c2patool will try to fetch the manifest for new_manifest.jpg from http://my_server/myasset.c2pa during validation.
+
+Note: It is possible to combine the `-s` and `-r` options. When used together a manifest will be embedded in the output files and the remote reference will also be added. The sidecar .c2pa file is omitted.
+
 ## Appendix
 
 ### Creating and using an X.509 certificate
```

Comment:
We should remove the section on generating certs until it is correct.  This caused an issued when someone tried this on Discord.

Reply:
Ok, I removed the whole appendix for now